src/ActionHandler/SynchronizeCollectionHandler.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace CommonGateway\VrijBRPToZGWBundle\ActionHandler;
+
+use App\ActionHandler\ActionHandlerInterface;
+use CommonGateway\VrijBRPToZGWBundle\Service\NewSynchronizationService;
+use CommonGateway\VrijBRPToZGWBundle\Service\VrijBrpService;
+
+class SynchronizeCollectionHandler implements ActionHandlerInterface
+{
+
+
+    /**
+     * @param NewSynchronizationService $synchronizationService
+     */
+    public function __construct(
+        private readonly NewSynchronizationService $synchronizationService,
```

Comment:
Avoid excessively long variable names like $synchronizationService. Keep variable name length under 20.

Reply:
Fixed in PR #4